php/commands/site.php

```diff
@@ -1403,6 +1403,82 @@ public function service_level($args, $assoc_args) {
   }
 
   /**
+   * Manage site organization tags
+   *
+   * ## OPTIONS
+   *
+   * <add|remove>
+   * : subfunction to run
+   *
+   * [--site=<site>]
+   * : Site's name
+   *
+   * [--org=<name|id>]
+   * : Organization to apply tag with
+   *
+   * [--tag=<tag>]
+   * : Tag to add or remove
+   *
+   * @subcommand tag
+   */
+  public function tag($args, $assoc_args) {
+    $action  = array_shift($args);
+    $site    = SiteFactory::instance(Input::sitename($assoc_args));
+    $org     = Input::orgid($assoc_args, 'org');
+    if ($site->get('organization') != null) {
```

Comment:
Thanks for the help! I went and put together a SiteOrganizationMembership class and its collection to simply manage this all and functionality.